databricks/sdk/runtime/__init__.py

```diff
@@ -6,19 +6,42 @@ def _get_global_dbutils():
         return None
 
 
-dbutils = _get_global_dbutils()
+is_oss_implementation = True
 
 try:
-    from .stub import *
+    # Internal implementation
+    from dbruntime import UserNamespaceInitializer
+
+    # All objects that are injected into the Notebook's user namespace should also be made
+    # available to be imported from databricks.sdk.runtime.globals. This import can be used
+    # in Python modules so users can access these objects from Files more easily.
+    __all__ = [
+        "display", "displayHTML", "dbutils", "table", "sql", "udf", "getArgument", "sc", "sqlContext", "spark"
+    ]
+
+    userNamespaceGlobals = UserNamespaceInitializer.getOrCreate().get_namespace_globals()
+    _globals = globals()
+    for var in __all__:
+        _globals[var] = userNamespaceGlobals[var]
+    is_oss_implementation = False
 except ImportError:
-    from databricks.sdk.dbutils import RemoteDbUtils
-    if not dbutils:
-        # this assumes that all environment variables are set
-        dbutils = RemoteDbUtils()
-except NameError:
-    from databricks.sdk.dbutils import RemoteDbUtils
-    if not dbutils:
-        # this assumes that all environment variables are set
-        dbutils = RemoteDbUtils()
+    # OSS implementation
+    is_oss_implementation = True
+    dbutils = _get_global_dbutils()
+
+    try:
+        from .stub import *
+    except ImportError:
+        from databricks.sdk.dbutils import RemoteDbUtils
+        if not dbutils:
+            # this assumes that all environment variables are set
+            dbutils = RemoteDbUtils()
+    except NameError:
+        from databricks.sdk.dbutils import RemoteDbUtils
```

Comment:
Why do we need to import remote dbutils implementation twice?…

Reply:
@nfx This is the original OSS implementation, but the commit suggestion by @kartikgupta-db down seems to solve this issue.